File: dotnet/src/Selenium.WebDriverBackedSelenium/Internal/SeleniumEmulation/MetaKeyDown.cs
using System;
using System.Collections.Generic;
using System.Text;
using OpenQA.Selenium;

namespace Selenium.Internal.SeleniumEmulation
{
    internal class MetaKeyDown : SeleneseCommand
    {
        private KeyState keyState;

        public MetaKeyDown(KeyState keyState)
        {
            this.keyState = keyState;
        }

        protected override object HandleSeleneseCommand(IWebDriver driver, string locator, string value)
        {
            keyState.MetaKeyDown = true;
            return null;
        }
    }
}

File: javasrc/org/openqa/selenium/internal/seleniumemulation/MetaKeyDown.java
package org.openqa.selenium.internal.seleniumemulation;

import org.openqa.selenium.WebDriver;

public class MetaKeyDown extends SeleneseCommand<Void> {
  private final KeyState keyState;

  public MetaKeyDown(KeyState keyState) {
    this.keyState = keyState;
  }

  @Override
  protected Void handleSeleneseCommand(WebDriver driver, String locator, String value) {
    keyState.metaKeyDown = true;
    return null;
  }
}